packages/contracts/test/core/dao/dao.ts

```diff
@@ -327,29 +331,43 @@ describe('DAO', function () {
   describe('Upgrades', async () => {
     let legacyContractFactory: ContractFactory;
     let currentContractFactory: ContractFactory;
+    let initArgs: any;
 
     before(() => {
       currentContractFactory = new DAO__factory(signers[0]);
+
+      initArgs = {
+        metadata: dummyMetadata1,
+        initialOwner: signers[0].address,
+        trustedForwarder: dummyAddress1,
+        daoURI: daoExampleURI,
+      };
     });
 
-    it('from v1.0.0', async () => {
+    it('upgrades to a new implementation', async () => {
+      await upgradeToSelfCheck(
+        signers[0],
+        signers[1],
+        undefined,
+        initArgs,
+        'initialize',
+        currentContractFactory,
+        UPGRADE_PERMISSIONS.UPGRADE_DAO_PERMISSION_ID
+      );
```

Comment:
Since the function as many args, i guess it would be a good practice for readablity to do:
1- add comment to the passed args:
`
someFunction(
  value1,  // explanation1
  value2,  // explanation2
  value3   // explanation3
);
`
or 
2- create a type for the function so the args. name could be visible:
`
function exampleFunction({ option1, option2, option3 }) {
  // use the options
}

exampleFunction({
  option1: value1,
  option2: value2,
  option3: value3
});
`